From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Sun, 29 May 2016 18:21:52 +0200
Subject: Set the icontheme for the unit_tests

Currently kpimtextedit uses media-playback-stop icons that are not
available in the hicolor icontheme (which is the default one for the
unit tests environment).
---
 src/composer-ng/autotests/richtextcomposertest.cpp     | 1 +
 src/texttospeech/autotests/texttospeechactionstest.cpp | 1 +
 src/texttospeech/autotests/texttospeechwidgettest.cpp  | 1 +
 3 files changed, 3 insertions(+)

diff --git a/src/composer-ng/autotests/richtextcomposertest.cpp b/src/composer-ng/autotests/richtextcomposertest.cpp
index a1c16d2..1a76716 100644
--- a/src/composer-ng/autotests/richtextcomposertest.cpp
+++ b/src/composer-ng/autotests/richtextcomposertest.cpp
@@ -43,6 +43,7 @@ RichTextComposerTest::RichTextComposerTest(QObject *parent)
     : QObject(parent)
 {
     qRegisterMetaType<KPIMTextEdit::RichTextComposer::Mode>();
+    QIcon::setThemeName(QStringLiteral("breeze"));
 }
 
 RichTextComposerTest::~RichTextComposerTest()
diff --git a/src/texttospeech/autotests/texttospeechactionstest.cpp b/src/texttospeech/autotests/texttospeechactionstest.cpp
index e327fa7..71ababc 100644
--- a/src/texttospeech/autotests/texttospeechactionstest.cpp
+++ b/src/texttospeech/autotests/texttospeechactionstest.cpp
@@ -29,6 +29,7 @@ TextToSpeechActionsTest::TextToSpeechActionsTest(QObject *parent)
     : QObject(parent)
 {
     qRegisterMetaType<KPIMTextEdit::TextToSpeechWidget::State>();
+    QIcon::setThemeName(QStringLiteral("breeze"));
 }
 
 TextToSpeechActionsTest::~TextToSpeechActionsTest()
diff --git a/src/texttospeech/autotests/texttospeechwidgettest.cpp b/src/texttospeech/autotests/texttospeechwidgettest.cpp
index 4070c13..b22a471 100644
--- a/src/texttospeech/autotests/texttospeechwidgettest.cpp
+++ b/src/texttospeech/autotests/texttospeechwidgettest.cpp
@@ -32,6 +32,7 @@ TextToSpeechWidgetTest::TextToSpeechWidgetTest(QObject *parent)
     : QObject(parent)
 {
     qRegisterMetaType<KPIMTextEdit::TextToSpeechWidget::State>();
+    QIcon::setThemeName(QStringLiteral("breeze"));
 }
 
 void TextToSpeechWidgetTest::addInterface(KPIMTextEdit::TextToSpeechWidget *widget)
